### PR TITLE
update OpenAPI output to use DisplayAttributes struct

### PR DESCRIPTION
**Pull request by madalynrose**

### CODE REVIEW

File: builtin/credential/ldap/path_users.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ func pathUsersList(b *backend) *framework.Path {
 
 		HelpSynopsis:    pathUserHelpSyn,
 		HelpDescription: pathUserHelpDesc,
+		DisplayAttrs: &framework.DisplayAttributes{
+			Navigation: true,
+		},
```
Comment (madalynrose) [inline builtin/credential/ldap/path_users.go]: @kalafut could the groups field to be a `CommaStringSlice` type?

Comment (madalynrose) [reply builtin/credential/ldap/path_users.go]: Also it’s likely intended but afaict there’s no validation on whether a group or policy exists when creating LDAP Groups & Users.

Comment (kalafut) [reply builtin/credential/ldap/path_users.go]: @madalynrose I think it could be converted. Lemme check

Comment (kalafut) [reply builtin/credential/ldap/path_users.go]: https://github.com/hashicorp/vault/pull/6942

**Pull request merged**